functions/package.json

```diff
@@ -12,6 +12,7 @@
     "firebase": "^8.2.4",
     "firebase-admin": "^9.4.2",
     "firebase-functions": "^3.13.1",
+    "lodash": "^4.17.21",
```

Comment:
@ermiaSparkle 
I think there is sense to make only lodash.chunk instead of all package, if only it's used

Reply:
fixed in 5de97cde144760c02d63d94cad7ef3f213a5d232